rllib/agents/maml/maml.py

```diff
@@ -0,0 +1,234 @@
+import logging
+
+import numpy as np
+from ray.rllib.utils.sgd import standardized
+from ray.rllib.agents import with_common_config
+from ray.rllib.agents.maml.maml_tf_policy import MAMLTFPolicy
+from ray.rllib.agents.trainer_template import build_trainer
+from typing import List
+from ray.rllib.evaluation.metrics import get_learner_stats
+from ray.rllib.execution.common import SampleBatchType, \
+    STEPS_SAMPLED_COUNTER, STEPS_TRAINED_COUNTER, LEARNER_INFO, \
+    _get_shared_metrics
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.execution.metric_ops import CollectMetrics
+from ray.util.iter import from_actors
+
+logger = logging.getLogger(__name__)
+
+# yapf: disable
+# __sphinx_doc_begin__
+DEFAULT_CONFIG = with_common_config({
+    # If true, use the Generalized Advantage Estimator (GAE)
+    # with a value function, see https://arxiv.org/pdf/1506.02438.pdf.
+    "use_gae": True,
+    # GAE(lambda) parameter
+    "lambda": 1.0,
+    # Initial coefficient for KL divergence
+    "kl_coeff": 0.0005,
+    # Size of batches collected from each worker
+    "rollout_fragment_length": 200,
+    # Stepsize of SGD
```

Comment:
This looks very large, no?

Reply:
True, Pendulum-Mass and Antrandgoal both have an episode horizon of length 200, hence it defaults to 200. (each worker env is stepping through 1 episode)